README.md

```diff
@@ -48,22 +47,32 @@ var TryStardust = React.createClass({
 module.exports = TryStardust;
 ```
 
-## Develop
+## Installation
+Start by cloning this repo and installing dependencies:
+
+    git clone git@github.com:TechnologyAdvice/stardust.git stardust && cd $_
```

Comment:
Agreed, not worth update if we aren't in agreement.  I brought it up as the README.ms is the first thing many folks will see of Stardust and I believe it sets the tone of the project.  I'll leave my peace here and move one :)

The parts of it I found strange were:
- The clone directory argument is unnecessary, it duplicates the repo name which is the default.  Why show how to duplicate arguments?
- There is a `&&` on line one, but the next command is on it's own line.  Why join some commands and not others?
- The combination of extra arguments, lesser known commands `$_`, and joining some commands and not others `&&`, and comment under this made it difficult to scan.  I had to scan it more than once.
- This README section of a well known project is typically pretty standard and I think consistency is good.

OK, with my thoughts out ^ I'm good moving on.